### PR TITLE
FIX: improve redirect page

### PR DESCRIPTION
**Pull request by jklymak**
This adds styling to the redirect page so when it flashes it at least doesn't look like an error..

### CODE REVIEW

**Comment by jklymak** [comment]
Second commit skips top-level index.html (ahem I think)

**Comment by jklymak** [comment]
ping @QuLogic and @tacaswell I think this would make our redirect page that flashes up for a couple of second look not so bad....

File: _websiteutils/make_redirects_links.py [hunk 1]
```diff
@@ -67,18 +66,38 @@ def findlast(fname, tocheck):
...
-<html lang="en">
-    <head>
-        <meta charset="utf-8">
-        <meta http-equiv="refresh" content="0;url={newurl}" />
-        <link rel="canonical" href="https://matplotlib.org/{canonical}" />
-    </head>
-    <body>
+html_redirect = """
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
+  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```
Comment (QuLogic) [inline _websiteutils/make_redirects_links.py:71]: Use html5 doctype, no need for xhtml at this point.

Comment (jklymak) [reply _websiteutils/make_redirects_links.py:71]: thats just from the top of matplotlib.org?
```
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

**Comment by jklymak** [comment]
Did we not want to do this?

File: _websiteutils/make_redirects_links.py (hunk 1 above)
Comment (QuLogic) [reply _websiteutils/make_redirects_links.py:71]: Not in the current theme: https://github.com/matplotlib/mpl-brochure-site/blob/gh-pages/index.html

**Comment by tacaswell** [comment]
@QuLogic Do we still use this script now that we have moved to selfhosting + caddy?

**Comment by QuLogic** [comment]
For the time being yes, there are no redirects in Caddy at the moment.

**Comment by QuLogic** [comment]
https://github.com/matplotlib/matplotlib.org/pull/41 took care of this.

**Pull request closed without merging**